source/_components/opnsense.markdown

```diff
@@ -0,0 +1,63 @@
+---
+title: "OPNsense"
+description: "Instructions on how to configure OPNsense integration"
+logo: opnsense.png
+ha_category:
+  - Hub
+  - Presence Detection
+ha_release: "0.100"
+---
+
+[OPNsense](https://opnsense.org/) is an open source HardenedBSD based firewall
+and routing platform. There is currently support for the following device types
+within Home Assistant:
+
+- [Presence Detection](#presence-detection)
+
+## Configuration
+
+To configure OPNsense integration with Home Assistant add the following section
+to your configuration.yaml:
+
+```yaml
+opnsense:
+  url: http://router/api
+  api_secret: APIsecret
+  api_key: apiKEY
+```
+
+Where the `api_key` and `api_secret` values are aquired from your OPNsense
+router using the web interface. For more inforation on this procedure refer
+to the OPNsense [documentation](https://docs.opnsense.org/development/how-tos/api.html#creating-keys)
+
+{% configuration %}
+url:
+  description: The url for the OPNsense api endpoint of your router
+  type: string
+  required: true
+api_key:
+  description: The API key used to authenticate with your OPNsense API endpoint
+  type: string
+  required: true
+  default: None
```

Comment:
Also here

Reply:
Oops, heh I guess I applied your earlier suggestion a bit too mechanically. Fixed in: https://github.com/home-assistant/home-assistant.io/pull/10427/commits/ddaab6b128d0a1c6376d13cebc22ec5cffa1eb22